staging/src/github.com/labring/image-cri-shim/pkg/cri/cri_test.go

```diff
@@ -26,5 +26,5 @@ func TestNewContainerRuntime(t *testing.T) {
 	//driver,_:=cr.processConfigFile()
 	//t.Log(driver)
 	cc := []string{"aa", "bb", "cc"}
-	println(fmt.Sprintf("%+v", cc))
+	fmt.Println(fmt.Sprintf("%+v", cc))
```

Comment:
Replace dupicate usage of  printf.
```suggestion
	fmt.Println("%+v", cc)
```

Reply:
done